docs/plugins/find-and-install-plugin.md

```diff
@@ -0,0 +1,26 @@
+(find-and-install-plugins)=
+
+# Finding and installing a napari plugin
+
+You can find plugins to use with napari on the [napari hub](https://napari.org),
+and can install plugins directly from your instance of napari itself.
+
+## Finding plugins on the napari hub
+
+The napari hub hosts information about all napari-compatible plugins.
+You can browse, search, and filter to find plugins that fit your needs.
+You can also share links to specific search results and plugins.
+
+## Installing plugins with napari
+
+All plugins listed on the napari hub can be installed directly from napari:
```

Comment:
"All packages on PyPI tagged with napari framework" (or something like it) feels a bit more accurate here... or are we generally trying to avoid mentioning pypi?

Reply:
Yeah, if we can have the introduction paragraph above, I would say here, "All packages annotated with the `Framework :: napari` tag can be installed directly from napari."

We might eventually change this if we decide to change the plugin discovery logic within napari, but for now I agree that listing the tag here makes more sense.